pages/post/[interviewId].js

```diff
@@ -85,3 +154,63 @@ const ThumbUpIcon = styled(ThumbUps)`
   fill: #95091b;
   margin-left: 20px;
 `;
+const CommentsContainer = styled.div`
+  margin-top: 20px;
+  margin-bottom: 200px;
+`;
+const InputContainer = styled.div`
+  margin-bottom: 10px;
+`;
+const Label = styled.label`
+  display: block;
+`;
+const TextInput = styled.input`
+  width: 100%;
+  padding: 5px;
+  font-size: 16px;
+  border-radius: 5px;
+  border: 1px solid #ccc;
+  outline: none;
+`;
+const TextArea = styled.textarea`
+  width: 100%;
+  padding: 5px;
+  font-size: 16px;
+  border-radius: 5px;
+  border: 1px solid #ccc;
+  outline: none;
+  resize: vertical;
+`;
+const ButtonContainer = styled.div`
+  text-align: right;
+  margin-bottom: 20px;
+`;
+const SubmitButton = styled.button`
+  padding: 10px 20px;
+  font-size: 16px;
+  border-radius: 5px;
+  background-color: #95091b;
+  margin-top: 10px;
+  border: none;
+  color: #fff;
+  cursor: pointer;
+  transition: background-color 0.2s ease-in-out;
+  &:hover {
+    background-color: #731c28;
+  }
+`;
+const Comment = styled.div`
+  margin-bottom: 20px;
+  border: 1px solid #95091b;
+`;
+const CommentName = styled.div`
+  font-weight: bold;
+  margin-bottom: 5px;
+`;
+const CommentMessage = styled.div`
+  white-space: pre-wrap;
+`;
+const CharCount = styled.div`
+  font-size: 12px;
+  margin-top: 5px;
+`;
```

Comment:
Now would be a good time to outsource these styled components to their own file =)
That way you will have a leaner component.
Don't forget to import them, though!